### DISCUSSION TITLE
Firmware upgrade of original AzulSCSI labeled board

### DISCUSSION THREAD

**Discussion by Fragmax2018** [ZuluSCSI/ZuluSCSI-firmware#10] [Q&A] [answered]
I have an original AzulSCSI labeled board v1.1 .  I ran first firmware upgrade and realized I needed to change the Zulu to Azul to get it to update the firmware after the project's name changed.  At that time my log files changed to A ZuluSCSI from AzulSCSI.  I went to update to 1.0.2 and nothing happed. I again needed to take the ZULU out and replace it with AZUL so the firmware would update.  Should I expect this will need to be done every time? or will a future firmware upgrade address this?

**Comment by PetteriAimonen** [comment, marked as answer]
We are investigating ways to upgrade the bootloader automatically in a future firmware upgrade.

In the meanwhile, it is possible to upgrade the bootloader to ZuluSCSI version by using USB cable, see instructions here:
https://github.com/ZuluSCSI/ZuluSCSI-firmware/issues/8
